thirdparty/README.md

```diff
@@ -481,8 +481,9 @@ Collection of single-file libraries used in Godot components.
   * Upstream: http://wss.co.uk/pinknoise/yuv2rgb/ (to check)
   * Version: ?
   * License: BSD
-
-
+- `smolv.h`
+  * Upstream: https://github.com/aras-p/smol-v
+  * License: Public Domain
 ## nanosvg
```

Comment:
I think you miss a few newlines here. Sorry, for this little cavil, great work :S

Reply:
Fixed.